src/main/java/Duke.java

```diff
@@ -1,10 +1,375 @@
+        } else{
+            completed = false;
+            //taskDescriptionOnly = userInput.split(" ", 4)[3];
+        }
+        taskDescriptionOnly = userInput.substring(userInput.indexOf(" ", 7), userInput.indexOf("/"));
+        Deadline deadLineTask = new Deadline(taskDescriptionOnly,
+                completed,identifyDeadlineCommand(userInput)[1]);
+        taskList.addTasks(deadLineTask);
+        deadLineTask.initialiseDeadline();
+    }
+
+    /**
+     * Creates a new output file if there is no saved file
+     *
+     * @param taskList
+     */
+    public static void createNewOutputFile(TaskList taskList) {
+        try {
+            String dir = System.getProperty("user.dir");
+            BufferedWriter bufferedWriter = new BufferedWriter(
+                    new FileWriter(dir + "\\duke.txt"));
+            for (int i = 0; i < taskList.savedTasks.size(); i++) {
+                bufferedWriter.write(taskList.savedTasks.get(i).toString() + "\n");
+            }
+            bufferedWriter.close();
+        } catch(IOException e){
+            return;
+        }
+    }
+
+    /**
+     * Adds a new todo task into the list of task.
+     * Displays that the task has been added into the list.
+     * @param taskList
+     * @param userInput
+     */
+    private static void addToDoTask(TaskList taskList, String userInput) {
+        boolean completed = false;
+        if(userInput.contains("[X]")){
+            completed = true;
+        }  else {
+            completed = false;
+        }
+        ToDo toDoTask = new ToDo(userInput.substring(userInput.indexOf(' ',0))
+                ,completed);
+        taskList.addTasks(toDoTask);
+        toDoTask.initialiseToDo();
+    }
+
+    /**
+     * Adds a new event task into the list of task.
+     * Displays that the task has been added into the list.
+     * @param taskList
+     * @param userInput
+     */
+    private static void addEventTask(TaskList taskList, String userInput) {
+        boolean completed = false;
+        if(userInput.contains("[X]")){
+            completed = true;
+        }  else {
+            completed = false;
+        }
+        Events eventTask = new Events(
+                userInput.substring(userInput.indexOf(' ',0), userInput.indexOf('/')),
+        completed,"/" + identifyDeadlineCommand(userInput)[1]);
+        taskList.addTasks(eventTask);
+        eventTask.initialiseEvent();
+    }
+
+    /**
+     * Adds a new deadline task into the list of task.
+     * Displays that the task has been added into the list.
+     * @param taskList
+     * @param userInput
+     */
+    private static void addDeadlineTask(TaskList taskList, String userInput) {
+        boolean completed = false;
+        if(userInput.contains("[X]")){
+            completed = true;
+        }  else {
+            completed = false;
+        }
+        Deadline deadLineTask = new Deadline(
```

Comment:
Good job in attempting to maintain the <120 characters per line standard. Where possible, try to avoid indicating multi-steps in a single line.